### PR TITLE
Add support for PHP 8.1

### PR DESCRIPTION
**Pull request by arueckauer**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

PHP 8.1 support.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,15 +23,15 @@
...
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
         "laminas/laminas-diactoros": "^1.7 || ^2.0",
         "laminas/laminas-http": "^2.11",
         "laminas/laminas-zendframework-bridge": "^1.0",
         "psr/http-message": "^1.0"
     },
     "require-dev": {
         "laminas/laminas-coding-standard": "~1.0.0",
-        "phpunit/phpunit": "^9.4.1"
+        "phpunit/phpunit": "^9.5.10"
```
Comment (Ocramius) [inline composer.json:34]: I think we should enable PHPUnit deprecation tracking here?

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (arueckauer) [reply composer.json:34]: Can you help me understand, how that works? Is that done in the PHPUnit configuration?

Comment (Ocramius) [reply composer.json:34]: Yes, see https://github.com/laminas/laminas-router/blob/44759e71620030c93d99e40b394fe9fff8f0beda/phpunit.xml.dist#L6

**Comment by arueckauer** [comment]
@froschdesign Do you know what the issue with markdownlint is and how to solve it? Your expertise is greatly appreciated.

**Comment by froschdesign** [comment]
@arueckauer 
For [`index.md`](https://github.com/laminas/laminas-psr7bridge/blob/1.4.x/docs/book/index.md?plain=1) use:

```markdown
<!-- markdownlint-disable -->
../../README.md
```

In the `intro.md` file add a new line at the end of the file:

```markdown
# Introduction

Code for converting [PSR-7](http://www.php-fig.org/psr/psr-7/) messages to
[laminas-http](https://docs.laminas.dev/laminas-http/) messages, and vice
versa.
<!-- new line here
```

And you can remove the unused `index.html` file.

**Review by Ocramius — APPROVED** [review]
LGTM :ship: 

Thanks @arueckauer!

**Pull request merged**